protobuf/lib/src/protobuf/builder_info.dart

```diff
@@ -219,10 +219,8 @@ class BuilderInfo {
   }
 
   // Map field.
-  void m<K, V>(int tagNumber, String name,
+  void m<K, V>(int tagNumber, String name, int keyFieldType, int valueFieldType,
```

Comment:
To avoid breaking the protobuf version we could keep the types named.
```
void m<K, V>(int tagNumber, String name,
     {String? entryClassName,
      required int keyFieldType,
      required int valueFieldType,
      CreateBuilderFunc? valueCreator,
      ValueOfFunc? valueOf,
      List<ProtobufEnum>? enumValues,
```

Because we never generated missing values here, and this api is only intended for generated code I would argue that that is non-breaking

Reply:
Done.